apps/bfd-server-ng/src/main/java/gov/cms/bfd/server/ng/util/LogUtil.java

```diff
@@ -0,0 +1,21 @@
+package gov.cms.bfd.server.ng.util;
+
+import org.slf4j.LoggerFactory;
+
+/** Utility class used for logging. */
+public class LogUtil {
+  private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(LogUtil.class);
+
+  /**
+   * For logging bene_sk.
+   *
+   * @param beneSk the bene_sk
+   */
+  public static void logBeneSkIfPresent(Long beneSk) {
```

Comment:
`IfPresent` is redundant in the method name now since it's not checking if it's present.
```suggestion
  public static void logBeneSk(Long beneSk) {
```